### ISSUE TITLE
FOIA and Section 508 of the Rehabilitation Act

### ISSUE THREAD

**Issue by philipashlock** [GSA/participate-nap4#128]
*This suggestion was submitted by Demand Progress. It also is [available online](https://medium.com/demand-progress/the-2017-opengov-national-action-plan-1d2ad8614a46).*

# Access to Information

## FOIA and Section 508 of the Rehabilitation Act

There is significant confusion around the extent to which Section 508 of the Rehabilitation Act prohibits agencies from posting information online. A memorandum or other document that details how the law is intended to work should be released to the public. In addition, DOJ should investigate the extent to which Section 508 is preventing the online publication of documents, what steps are necessary to remove roadblocks, and how agencies may collaborate with the public to meet the needs of full disclosure as well as the legal requirements.

cc @DanielSchuman
